.github/actions/multi-approvers/tests/github-main.test.ts

```diff
@@ -0,0 +1,176 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import assert from "node:assert/strict";
+import { mock, test } from "node:test";
+import * as ghCore from "@actions/core";
+import { context as ghContext } from "@actions/github";
+import { main } from "../src/github-main";
+
+type Core = typeof ghCore;
+type Context = typeof ghContext;
```

Comment:
Why?

Reply:
Oh, and these types are not exported, so the only way to get them is to use the `typeof` operator on the exported value.

Reply:
I think this is bringing Java into JavaScript unnecessarily; we don't need dependency injection.

But also, we don't need the entire construct to inject: we only need a subset of the functions in `actions/core`, and `actions/context` is just a JSON schema.